client/coral-admin/src/graphql/queries/index.js

```diff
@@ -62,13 +63,21 @@ export const loadMore = (fetchMore) => ({limit, cursor, sort, tab, asset_id}) =>
       asset_id
     },
     updateQuery: (oldData, {fetchMoreResult:{comments}}) => {
-      return {
+      const updatedData = {
         ...oldData,
         [tab]: [
           ...oldData[tab],
           ...comments
         ]
       };
+
+      // if we're not in the all tab, put the new comments in there anyway
+      // this way the acceptComment and rejectComment mutations don't break
```

Comment:
I think this comments should be more like: `Always put comments into the all tab to maintain a consistent state.`. The breaking mutations is a result of an inconsistent state.